pyproject.toml

```diff
@@ -98,6 +98,7 @@ omit = [
 line-length = 140
 # F821 needed to avoid false-positives in nested functions, F722 due to jaxtyping 
 lint.ignore = ["F821", "F722"]
+lint.select = ["NPY201"]
```

Comment:
It was tested locally. This works on the example from [here](https://docs.astral.sh/ruff/rules/numpy2-deprecation/#numpy2-deprecation-npy201).